setup.py

```diff
@@ -20,7 +20,7 @@
     'enum34; python_version < "3.4"',
     'google-auth-oauthlib>=0.0.1,<1.0.0',
     'google-api-core[grpc] >= 1.1.0, < 2.0.0dev',
-    'PyYAML >= 3.11, < 4.0',
+    'PyYAML >=4.2b1',
```

Comment:
As an FYI, a lack of an upper-bound has burned us before. :-)

Generally speaking, a major version bump is probably a good point to work with, as those are expected to include breaking changes.

Reply:
Good idea - added.